### PR TITLE
Dashboard card

### PR DESCRIPTION
**Pull request by mylescarey2019**
New Route/View for Events that are assigned to the current user.  Added link in SideNav and supporting code in Dashboard, event.js, eventController.js and API.js

### CODE REVIEW

File: client/src/components/SideNav/index.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ class SideNav extends Component {
         <div id="filterDivider">Filters</div>
         <button> <a href='/dashboard'>All Household Events</a> </button>
         <button> <a href='/dashboard?view=myevents'>My Events</a> </button>
+        <button> <a href='/dashboard?view=assigned'>Events Assigned To Me</a> </button>
```
Comment (jennhenn18) [inline client/src/components/SideNav/index.js]: Can we remove "Events" before "Assigned to Me"

File: controllers/eventController.js [hunk 2]
```diff
@@ -57,6 +57,24 @@ module.exports = {
     });
   }, 
   
+  // Return events assigned to the user for the given household and populate member references
+  // match: {assigneeStatus: "claimed"},
+  //, match: {assignedStatus: "unclaimed"}
+  findUserAssignedPopulated: function(householdId,userId) {
+    return new Promise((resolve, reject) => {
+      db.Household.find({ _id: householdId },{members:0,name:0,_id:0,__v:0,invitees:0 })
```
Comment (jennhenn18) [inline controllers/eventController.js]: What are the 0's doing here?

Comment (mylescarey2019) [reply controllers/eventController.js]: the 0s tell mongoose to not return the given field from the collection.  These are unnecessary fields for this particular query

File: client/src/components/SideNav/index.js (hunk 1 above)
Comment (mylescarey2019) [reply client/src/components/SideNav/index.js]: yes that is a good suggestion

File: controllers/eventController.js (hunk 2 above)
Comment (jennhenn18) [reply controllers/eventController.js]: Sweet! Good to know.

**Pull request merged**